index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Cuau Cabrera - FullStack Developer</title>
+</head>
+
+<body>
+    <div class="headline-container">
+        <!-- Header and navbar -->
+        <header class="header">
+            <nav>
+                <img src="./assets/Toolbar - Dark - Compact.png" alt="">
+            </nav>
+        </header>
+        <!-- Hero headline -->
+        <section class="headline">
+            <h1 class="headline__title"><span class="headline__title--black">Hey there. <br>I'm Cuau. <br><span
+                        class="headline__title--green">I'm a
+                        software developer</span></span></h1>
+            <p class="headline__text"> I can help you build a product , feature or website. Look through some of my work
+                and experience! If you like what you see and have a project you need coded,
+                don’t hesitate to contact me. </p>
+            <div class="headline__icons">
+                <ul>
+                    <li><a href="https://twitter.com/cuau_cabrera"><img src="./assets/twitter.png"
+                                alt="This is a Twitter icon. Click on it to visit my Twitter profile"
+                                class="headline__icons--twitter"></a></li>
+                    <li><a href="https://linkedin.com/in/cuaucabrera"><img src="./assets/Linkedin icon.png"
+                                alt="This is a Linkedin icon. Click on it to visit my Linkedin profile"
+                                class="headline__icons--linkedin"></a></li>
+                    <li><a href="https://github.com/cuauh-cabrera"><img src="./assets/github.png"
+                                alt="This is a github icon. Click on it to visit my github profile"></a></li>
```

Comment:
> - [ ] Great progress so far!👍 Based on the design template, there are five social icon images, but you currently have only three. It would be greatly appreciated if you could include the missing icons from the design template in order to closely follow the Figma design. By following the Figma design closely, you ensure consistency throughout the development process.😊
> 
> Figma design:
> 
> ![desc1](https://github.com/cuauh-cabrera/my_portfolio/assets/40009354/dc612109-f5ca-4111-927f-f0c00efa6cc6)
> 
> Your design:
> 
> ![des](https://github.com/cuauh-cabrera/my_portfolio/assets/40009354/b81a687d-88a6-4ad2-be77-0be409c552a3)